src/components/books.js

```diff
@@ -0,0 +1,7 @@
+function Books() {
+  return (
+        <h1></h1>
```

Comment:
- [ ] It'd be great if books.js show a list of books, please work on it. 🐶 
- [ ] Please, create a book component for each book, these books could be listed on books.js according to the project requirements.